skorch/tests/test_net.py

```diff
@@ -2983,6 +2996,74 @@ def test_save_load_state_dict_custom_module(
         weights_loaded = net_new.custom_.state_dict()['sequential.3.weight']
         assert (weights_before == weights_loaded).all()
 
+    def test_torch_load_kwargs_auto_weights_only_false_when_load_params(
+            self, net_cls, module_cls, monkeypatch, tmp_path
+    ):
+        # Here we assume that the torch version is low enough that weights_only
+        # defaults to False. Check that when no argument is set in skorch, the
+        # right default is used.
+        # See discussion in 1063
```

Comment:
I wonder if we should add a test that tests the default for known old PyTorch versions?
We don't know whether PyTorch 2.6 will have this default but we know that <=2.6.0 should have this behavior.

It is unlikely that we'll add such a test at a later point in time.

Reply:
Not sure I fully understand the suggestion. Would this amount to monkeypatching `torch.__version__` instead of `get_torch_load_kwargs`? I didn't want to write a test that one day suddenly fails because the torch versions in CI were updated.

Reply:
No, I meant more like a `skipif` that checks the pytorch version to make sure that functionality is coupled with the pytorch version in the environment.

Reply:
Okay, I think I got it. The test was added: https://github.com/skorch-dev/skorch/pull/1064/commits/ab9c536516466f2fdb029caf71fd43f2d6e5d375